drfuzz/drfuzz_mutator.c

```diff
@@ -208,8 +208,8 @@ static bool
 drfuzz_parse_dictionary(const char *file, mutator_t *mutator)
```

Comment:
Also, this fixes #1928, right?  So instead of `Issue: #1928` it should have `Fixed #1928` on a line by itself.